src/encoded/schemas/mixins.json

```diff
@@ -615,6 +615,16 @@
         "lab.display_title": {
             "title": "Lab"
         }
+    },
+    "facets_aggregated_badges" : {
+        "aggregated_items.badges.item.badge.positive_badge": {
+            "title": "Positive Badges",
+            "weight": 997
+        },
+        "aggregated_items.badges.item.badge.warning_badge": {
+            "title": "Warning Badges",
+            "weight": 998
```

Comment:
Did you want to rename "weight" to "order"? Alex's column changes also uses "order" in the same way.